keyboards/planck/keymaps/ipetepete/keymap.c

```diff
@@ -0,0 +1,243 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _GAM3R 2
+#define _SPCFN 3
+#define _LOWER 4
+#define _RAISE 5
+#define _GAMUP 8
+#define _FUNC 10
+#define _MOUS 12
+#define _ADJUST 16
```

Comment:
We recommend using enums for layer names, as they are easier to manage long term, and auto-increment. 
```suggestion
enum layers {
    _QWERTY,
    _COLEMAK,
    _GAM3R,
    _SPCFN,
    _LOWER,
    _RAISE,
    _GAMUP,
    _FUNC,
    _MOUS,
    _ADJUST,
};
```